README.md

```diff
@@ -78,6 +80,58 @@ This process can be largely accelerated by using [cachix](https://cachix.org):
 cachix use jupyterwith
 ```
 
+### Using Flakes
+
+With Nix flakes (experimental Nix feature) you can automatically pin and update jupyterWith to the latest version.
+An example `flake.nix` file for jupyterWith on x86-64 Linux, executable with `nix run` from the same folder, looks like this:
+
+```
+{
+  description = "JupyterLab Flake";
+
+  inputs = {
+      jupyterWith.url = "github:tweag/jupyterWith";
+  };
+
+  outputs = { self, nixpkgs, jupyterWith }:
+    let
+      notebooks = ./.;
+
+      pkgs = import nixpkgs {
+        system = "x86_64-linux";
+        overlays = nixpkgs.lib.attrValues jupyterWith.overlays;
+        config = {
+          allowUnfree = true;
+        };
+      };
+      
+      iHaskell = jupyter.kernels.iHaskellWith {
+        name = "haskell";
+        packages = p: with p; [ hvega ];
+      }; 
+
+      iPython = pkgs.kernels.iPythonWith {
+        name = "Python-env";
+        packages = p: with p; [ sympy numpy ];
+        ignoreCollisions = true;
+      };
+
+      jupyterEnvironment = pkgs.jupyterlabWith {
+          kernels = [ iPython iHaskell ];
+      };
+    in
+    {
+#      packages.x86_64-linux.jupyterLab = jupyterEnvironment;
+#      defaultPackage.x86_64-linux = jupyterEnvironment;
```

Comment:
What are these supposed to do?

Reply:
They are for `nix build`, but yea they should probably be uncommented (or removed, couldn't decide).

Reply:
I think the best way is to remove this in the README flake to keep it short, but keep it in [example/flake/flake.nix](https://github.com/tweag/jupyterWith/blob/master/example/flake/flake.nix), and mention this file in the README.md as the full flake.